Traceback (most recent call last):
  File "/home/ltnghia02/vischronos/scripts/first.py", line 32, in <module>
    logging.basicConfig(filename='/home/ltnghia02/logs/script_execution1.log', level=logging.INFO,
  File "/usr/lib/python3.10/logging/__init__.py", line 2040, in basicConfig
    h = FileHandler(filename, mode,
  File "/usr/lib/python3.10/logging/__init__.py", line 1169, in __init__
    StreamHandler.__init__(self, self._open())
  File "/usr/lib/python3.10/logging/__init__.py", line 1201, in _open
    return open_func(self.baseFilename, self.mode,
FileNotFoundError: [Errno 2] No such file or directory: '/home/ltnghia02/logs/script_execution1.log'
